src/main/java/net/snowflake/ingest/streaming/internal/AbstractRowBuffer.java

```diff
@@ -673,15 +671,6 @@ static <T> AbstractRowBuffer<T> createRowBuffer(
     }
   }
 
-  private void checkBatchSizeEnforcedMaximum(float batchSizeInBytes) {
```

Comment:
Should this still be here but with higher limits?

Reply:
+1

Reply:
Sorry, I somehow missed this PR until it got merged. I have concerns about this change because there was another reason why these limits were introduced - to prevent long GC pauses. We observed JVM freezes for customers who were passing hundreds of thousands of rows into `insertRows()`, see more details [here](https://snowflakecomputing.atlassian.net/browse/SNOW-861190?focusedCommentId=3486934).

I would prefer to keep some limit there, customers should not pass hundreds of megabytes of data in a single batch. 

cc @sfc-gh-wfateem

Reply:
@sfc-gh-alhuang Do you want to run some local tests to see if things are working with a few GBs of a batch and 1 minutes of flush interval? If not, then we can add the exception back but with a much higher limit, WDYT?

Reply:
Got GCLocker timeout error message after building a 600+MB of blobs. I'll add the assertion back with higher limit.